pyscriptjs/tests/integration/support.py

```diff
@@ -746,7 +905,11 @@ def _router(self, route):
             assert url.path[0] == "/"
             relative_path = url.path[1:]
             if os.path.exists(relative_path):
-                route.fulfill(status=200, path=relative_path)
+                headers = {
+                    "Cross-Origin-Embedder-Policy": "require-corp",
+                    "Cross-Origin-Opener-Policy": "same-origin",
+                }
+                route.fulfill(status=200, headers=headers, path=relative_path)
             else:
                 route.fulfill(status=404)
```

Comment:
perhaps you can supply headers here as well?

Reply:
why?